src/dotnet/Chat.UI.Blazor/Components/ContactListNavbarWidget.razor

```diff
@@ -130,7 +130,7 @@
             case ContactGroup.Favorites:
```

Comment:
Not related to this PR, but IMO we should either have pins or favorites. Let's talk about this once AK is back & decide what's better here.